src/server/apsystem.ts

```diff
@@ -480,6 +507,19 @@ export default class ActivityPubSystem {
     await this.sendTo(followerURL, fromActor, response)
   }
 
+  async repliesCollection (fromActor: string, inReplyTo: string): Promise<APCollection> {
+    const items = await this.store.forActor(fromActor).inboxObjects.list({ inReplyTo })
+    const id = this.makeURL(`/v1/${fromActor}/inbox/replies/${encodeURIComponent(inReplyTo)}`)
+
+    return {
+      '@context': 'https://www.w3.org/ns/activitystreams',
+      type: 'Collection',
+      id,
+      items,
```

Comment:
as i understand, this will be an array of objects, right? do we need to store and send objects, or we could just store their ids and have clients dereference them? remote instances may have their own policies on who can see these.

mastodon for instance returns an array of ids:

```json
{
  "@context": "https://www.w3.org/ns/activitystreams",
  "id": "https://INSTAN.CE/users/USER/statuses/NUMERIC_ID/replies?only_other_accounts=true&page=true",
  "type": "CollectionPage",
  "partOf": "https://INSTAN.CE/users/USER/statuses/NUMERIC_ID/replies",
  "items": [
    "https://..."
  ]
}
```

Reply:
I'm betting mastodon does that to so save on bandwidth when rendering the outbox. When I visit the replies collection directly on my instance it shows the full objects. https://mastodon.mauve.moe/users/mauve/statuses/112566153123803805/replies